modules/core/arrow-validation/src/main/kotlin/arrow/validation/refinedTypes/bool/Not.kt

```diff
@@ -0,0 +1,16 @@
+package arrow.validation.refinedTypes.bool
+
+import arrow.Kind
+import arrow.validation.Refinement
+
+interface Not<F, B> : Refinement<F, B> {
+
+  fun REF(): Refinement<F, B>
+
+  override fun B.refinement(): Boolean = REF().run { !refinement() }
+
+  fun B.not(): Kind<F, B> = refine(this)
```

Comment:
Should we make these `operator`?

Reply:
Do you mean infix functions? I can overload the not operator which I suppose it would be `!` but I cannot overload `greater` as an operator 🤔